subsys/bluetooth/services/cgms/cgms.c

```diff
@@ -0,0 +1,451 @@
+	case 1:
+	case 3:
+	case 5:
+	case 7:
+	case 8:
+	case 10:
+		sst.day = 31;
+		break;
+	case 4:
+	case 6:
+	case 9:
+	case 11:
+		sst.day = 30;
+		break;
+	}
+	return len;
+}
+
+static ssize_t read_status(struct bt_conn *conn, const struct bt_gatt_attr *attr,
+			void *buf, uint16_t len, uint16_t offset)
+{
+	struct net_buf_simple *status_buf = NET_BUF_SIMPLE(5);
+
+	net_buf_simple_init(status_buf, 0);
+
+	status.time_offset = (k_uptime_get_32() - local_start_time) / 60;
+	net_buf_simple_add_le16(status_buf, status.time_offset);
+	net_buf_simple_add_u8(status_buf, status.status.status);
+	net_buf_simple_add_u8(status_buf, status.status.calib_temp);
+	net_buf_simple_add_u8(status_buf, status.status.warning);
+
+	return bt_gatt_attr_read(conn, attr, buf, len, offset,
+				status_buf->data, status_buf->len);
+}
+
+static void bt_cgms_notify_meas(struct bt_conn *conn, void *data)
+{
+	struct bt_cgm_meas_t *meas = (struct bt_cgm_meas_t *)data;
+
+	struct net_buf_simple *meas_buf = NET_BUF_SIMPLE(15);
+
+	net_buf_simple_init(meas_buf, 1);
+
+	net_buf_simple_add_u8(meas_buf, meas->flag);
+	net_buf_simple_add_le16(meas_buf, meas->glucose_concentration);
+	net_buf_simple_add_le16(meas_buf, meas->time_offset);
+	if (meas->sensor_status_annunciation.status != 0) {
+		net_buf_simple_add_u8(meas_buf, meas->sensor_status_annunciation.status);
+	}
+	if (meas->sensor_status_annunciation.calib_temp != 0) {
+		net_buf_simple_add_u8(meas_buf, meas->sensor_status_annunciation.calib_temp);
+	}
+	if (meas->sensor_status_annunciation.warning != 0) {
+		net_buf_simple_add_u8(meas_buf, meas->sensor_status_annunciation.warning);
+	}
+
+	uint8_t meas_size;
+
+	meas_size = meas_buf->len + 1;
+	net_buf_simple_push_u8(meas_buf, meas_size);
+
+	/* If conn is NULL, it implies this is a periodic notification.
+	 * Send it to all peers.
+	 */
+	if (conn == NULL) {
+		bt_gatt_notify(conn, &cgms_svc.attrs[CGMS_SVC_MEAS_ATTR_IDX],
+			meas_buf->data, meas_size);
+		return;
+	}
+	/* If conn is set, verify if the notification of this conn is enabled,
+	 * and send notification accordingly.
+	 */
+	bool notify_enabled;
+
+	notify_enabled = bt_gatt_is_subscribed(conn,
+		&cgms_svc.attrs[CGMS_SVC_MEAS_ATTR_IDX], BT_GATT_CCC_NOTIFY);
+
+	LOG_DBG("%s", notify_enabled ? "enabled" : "disabled");
+	/* If it is enabled, send the notification. */
+	if (notify_enabled) {
+		bt_gatt_notify(conn, &cgms_svc.attrs[CGMS_SVC_MEAS_ATTR_IDX],
+			meas_buf->data, meas_size);
+	}
+}
+
+static ssize_t read_session_run_time(struct bt_conn *conn, const struct bt_gatt_attr *attr,
+			void *buf, uint16_t len, uint16_t offset)
+{
+	return bt_gatt_attr_read(conn, attr, buf, len, offset, &srt, sizeof(srt));
+}
+
+static ssize_t racp_on_receive(struct bt_conn *conn,
+			const struct bt_gatt_attr *attr,
+			const void *buf,
+			uint16_t len, uint16_t offset, uint8_t flags)
+{
+	int rc;
+
+	rc = cgms_racp_recv_request(conn, buf, len);
+
+	if (rc < 0) {
+		LOG_WRN("Internal Error during RACP Handling: %d", rc);
+	}
```

Comment:
function call and check of return value is a single code block and should not have a blank line in between

```suggestion
	rc = cgms_racp_recv_request(conn, buf, len);
	if (rc < 0) {
		LOG_WRN("Internal Error during RACP Handling: %d", rc);
	}
```